### DISCUSSION TITLE
Welcome to threat-feed Discussions!

### DISCUSSION THREAD

**Discussion by winromulus** [emberstack/threat-feed#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by david-szymanski** [comment]
Hey there! I followed the steps in the guide you outlined for your FortiGate solution to ads but I'm still having trouble getting things to work. Is there any chance you could help me troubleshoot?
